GoogleCloudPlatform/anthos-service-mesh-samples
# Delete docs/migrate-istio-to-asm directory

### Background 
* See the [Istio-to-managed-ASM migration tutorial here](https://cloud.devsite.corp.google.com/service-mesh/docs/migrate-istio-to-anthos-service-mesh#test-using-canary-deployment).
* All versions of the tutorial have been updated in [this Google-internal Changelist](https://critique.corp.google.com/cl/496432278) such that they use the samples inside [`/docs/migrate-to-managed-asm/`](https://github.com/GoogleCloudPlatform/anthos-service-mesh-samples/pull/114/files).

### Change Summary
* Delete the unused `/docs/migrate-istio-to-asm/` directory.

### Testing Procedure
To be thorough, you can check that the [new samples in new folder](https://github.com/GoogleCloudPlatform/anthos-service-mesh-samples/pull/114/files) are _functionally_ identical.
